src/components/GeneralStatsCard.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="flex-wrap m-3 p-3">
+    <div class="flex flex-row">
+      <div class="flex-col ">
+        <font-awesome-icon
+          class="content-center mx-1 phone-icon"
+          icon="phone-alt"
+        />
+      </div>
+      <div class="flex-col">
+        <div class="justify-between flex flex-row">
+          <div class="" :peoplewaiting="peoplewaiting">{{ peoplewaiting }}</div>
+          <button class="text-sm">show all</button>
```

Comment:
I know this is just preliminary, but I would advise to always use the built components like `<base-button>`, as they already conform to the style guide and have some pre styling. 

Also just as an fyi, the application colors have all been added in `tailwind.config.js` so any colors you may need to use for styling will be in there.

Reply:
Thanks @tabiodun -- Have not had a moment yet to do a proper review for em'